### ISSUE TITLE
Guilty Gear Accet Core Plus[NPUB30957]: infinite verifying latest data with update 1.01

### ISSUE THREAD

**Issue by toccata10** [RPCS3/rpcs3#7676]
Archlinux, RPCS3 v0.0.9-9801-d94b8751
The psn base version works fine. But if I install the update 1.01 which brings the "Core Plus R" and if I accept the autosave, then there's a "verifying latest data" overlay, the game is dimmed (but it's still playable (but very dark)).
If I remove this folder, and restart, it works fine (as long as I don't save)
.config/rpcs3/dev_hdd0/home/00000001/savedata/NPUB30957-SYSTEM

So, the workaround is either to not update, or to never save.
My log file:
Guilty Gear Accet Core Plus[NPUB30957]: infinite verifying latest data with update 1.01

[RPCS3.log-guiltygear.gz](https://github.com/RPCS3/rpcs3/files/4279883/RPCS3.log-guiltygear.gz)

**Comment by toccata10** [comment]
![Capture_110](https://user-images.githubusercontent.com/7478580/75753318-cc8ca380-5d2a-11ea-96f2-1665e4cf099e.jpg)

**Comment by digitaldude555** [comment]
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/5938949/RPCS3.log.gz)
Remains an issue.

**Comment by faux123** [comment]
Yes, still an issue with latest build RPCS3 0.0.24-1-dd8a337b alpha

**Comment by Triticum0** [comment]
Retested with v0.0.30-15914-2a1af17f. No changes

**Comment by CreeperGate** [comment]
On the latest update I managed to fix it, at least once, by going to network.

**Comment by AniLeo** [comment]
@eandmsz bug reports from unofficial builds are not allowed

**Comment by digant73** [comment]
@Triticum0 is that still an issue?

**Comment by JasonShearer** [comment]
Still an issue as of today's date.

Edit: sukoshiotokonoko is right in that going to Network and returning to the main menu acts as a workaround.
